src/main/kotlin/io/bazel/worker/InvocationWorker.kt

```diff
@@ -23,7 +23,10 @@ class InvocationWorker(private val arguments: Iterable<String>) : Worker {
     runCatching {
       WorkerContext.run {
         doTask("invocation") { ctx -> execute(ctx, arguments) }.run {
-          println(log.out.toString())
+          val out = log.out.toString()
+          if (out != "") {
```

Comment:
The Kotlin standard library has `isEmpty` and `isBlank` functions on string types that does just this.

Reply:
(no longer applicable)